stock_demand_estimate/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 {
     "name": "Stock Demand Estimate",
     "summary": "Allows to create demand estimates.",
-    "version": "16.0.1.1.0",
+    "version": "17.0.1.1.0",
```

Comment:
@CarlosVForgeFlow Could you reset to 17.0.1.0.0 ?

Reply:
Done!